### PR TITLE
fix: update ingestion date field (MAPCO-2669)

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                       |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✔                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: src/update/metadataMerger.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { LayerMetadata } from '@map-colonies/mc-model-types';
 import { Footprint } from '@map-colonies/mc-utils';
 import { createBBoxString } from '../utils/bbox';
 import { layerMetadataToPolygonParts } from '../common/utills/polygonPartsBuilder';
+import { getUtcNow } from '../utils/getUtcDate';
```
Comment (asafMasa) [inline src/update/metadataMerger.ts:8]: as it is being used also in exporter - do you want to add it to utils and publish a new package?

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good Job! 👍

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "@map-colonies/mc-logger": "^2.0.2",
     "@map-colonies/mc-model-types": "^13.4.1",
     "@map-colonies/mc-probe": "^1.0.0",
-    "@map-colonies/mc-utils": "^1.5.0",
+    "@map-colonies/mc-utils": "file:../mc-utils/map-colonies-mc-utils-1.5.0.tgz",
```
Comment (asafMasa) [inline package.json:49]: fix to correct package

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/update/metadataMerger.ts (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/update/metadataMerger.ts]: done

File: package.json (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply package.json]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**